events/js/jquery.contentcarousel.js

```diff
@@ -0,0 +1,266 @@
+(function($) {
+	var	aux		= {

```

Comment:
@polybuildr , @hharchani , @nisargjhaveri 

HTML is indent 2 spaces. 
CSS and JS is indent 4 spaces. 

Or do you prefer something other than these conventions?